system-variables.md

```diff
@@ -462,6 +581,7 @@ Constraint checking is always performed in place for pessimistic transactions (d
 - Scope: SESSION | GLOBAL
 - Default value: -1
 - This variable is used to set the concurrency of the `hash join` algorithm.
+- A value of `-1` means that the value of `tidb_executor_concurrency` will be used instead.
```

Comment:
@morgo @XuHuaiyu 
Shall we add a note or warning to indicate it's deprecated in 5.0? For example: https://github.com/pingcap/docs/pull/4602/files#diff-ec2cbb115cc58fe3afe2862eb5148732d323cbf7ae35dea97bd381a1f8d0e8a0R458-R460

Reply:
Yes. It is the same case as tidb_hashagg_final_concurrency.

Reply:
I checked the file. There is already a depecation warning.